lib/mayaUsd/ufe/UsdAttributes.cpp

```diff
@@ -112,6 +112,11 @@ static PXR_NS::UsdAttribute _GetAttributeType(const PXR_NS::UsdPrim& prim, const
 
 Ufe::Attribute::Type UsdAttributes::attributeType(const std::string& name)
 {
+    // early return if name is empty.
+    if (name.empty()) {
+        throw std::invalid_argument("Empty name.");
```

Comment:
I'm not really familiar with how MayaUsd exceptions are thrown. Is this enough information, or do you need to somehow include where this is coming from?

Reply:
reverted.